src/pages/index.js

```diff
@@ -29,54 +32,153 @@ const enableValidation = (config) => {
 
 enableValidation(config);
 
-const createCard = (cardItem) => {
-  const card = new Card(cardItem, "#card-template", () => {
-    console.log(cardItem);
-    imgPopup.open(cardItem);
+//API
+
+export const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  authorization: "7dcd9a93-149c-4e56-87db-9285c9177a9e",
+});
+
+api
+  .getUserData()
+  .then((results) => results.json())
```

Comment:
```js

    }).then((res) => {
      if (res.ok) {
        return res.json();
      }
      return Promise.reject(`Error: ${res.status}`);
    });
```

According to the task you need to check the response if it's ok


And it should be done inside `Api`  - it's responsibility of that class

Reply:
fixed this